05.wc/wc.rb

```diff
@@ -3,66 +3,51 @@
 require 'optparse'
 
 def main
-  option_and_paths => { paths:, **option_flags }
-  display(paths, option_flags)
+  paths, target_options = options_and_paths
```

Comment:
`options_and_paths`なら、その順番に合わせて


```suggestion
  target_options, paths = options_and_paths
```

にする方がいいかも。

あと、`target_`は省略してもいいんじゃないでしょうか。

https://bootcamp.fjord.jp/pages/avoid-too-specific-var-name

Reply:
[返り値や引数の順番を修正](https://github.com/s-tone-gs/ruby-practices/pull/10/commits/d31e9b17ab8640e180ff8b68e5f9bab1cc4925be)にて修正しました！